openssh/README.md

```diff
@@ -0,0 +1,132 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+set -x
+
+# set environment from the last openssh build
+source results/last_build.env
+
+# stop the habitat supervisor
+hab sup term
+
+# add some required software before building the openssh_test
+hab pkg install -b core/hab-studio
+hab pkg install -b core/busybox
+hab pkg install -b chef/inspec
+
+# install the openssh artifact
+hab pkg install -b results/$pkg_artifact
+
+# restart the habitat supervisor and wait for it to start
+hab sup run &
+sleep 20
+
+# load the openssh habitat service
+hab svc load $pkg_ident
+set +x
+```
+
+Wait until the service has loaded successfully.  At this stage something like the following will appear in the terminal:
+
+```bash
+openssh.default(CF): Created configuration file /hab/svc/openssh/config/sshd_config
```

Comment:
If I use the steps I gave in my previous comment  then I don't get any of this output; but the tests all pass.
Can this output be removed from the instructions, or at least reduced to the lines that the reader _must_ watch out for when they're running through them?

Reply:
I've removed the steps for stopping/starting the supervisor and verified that the service loads fine.